docs/content/docs/user-docs/cross-region-resource-management.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Cross-Region Resource Management"
+description: "Managing resources across different AWS regions"
+lead: ""
+draft: false
+menu: 
+  docs:
+    parent: "installing"
+weight: 50
+toc: true
+---
+
+If you are using resources across different regions, you can override the default region of a given ACK service controller. ACK service controllers will first look for a region in the following order:
+
+  1. The region annotation `services.k8s.aws/region` on the resource. If provided, this will override the namespace default region annotation.
+  2. The namespace default region annotation `services.k8s.aws/default-region`.
+  3. Controller flags, such as the `aws.region` variable in a given Helm chart
+  4. Kubernetes pod [IRSA](https://aws-controllers-k8s.github.io/community/docs/user-docs/irsa/) environment variables
+
+For example, the ACK service controller default region is `us-west-2`. If you want to create a resource in `us-east-1`, use one of the following options to override the default region
```

Comment:
Instead of using the term "default region", I think it would be better to say "For example, the `--aws-region` ACK service controller flag is `us-west-2`. If you want to create..."